keyboards/mint60/readme.md

```diff
@@ -0,0 +1,15 @@
+# Mint60
+
+![Mint60](https://imgur.com/a/8xnTS4U)
+
+A short description of the keyboard/project
+
+Keyboard Maintainer: [Eucalyn](https://github.com/eucalyn)  [@eucalyn_](https://twitter.com/eucalyn_)
+Hardware Supported: The Mint60 PCBs, ProMicro supported  
+Hardware Availability: links to where you can find this hardware
```

Comment:
Do you have a link for this?  

Are you running this as a group buy, or selling the PCB/kit?  

So you plan to? 

If not, then this should go in /keyboards/handwire/ instead, as the main folder is meant more for production boards.

Reply:
I'm planning group buy and I will open PCBs & case data in GitHub soon.
But case data is incomplete now.